docs/CodeTransformation.md

```diff
@@ -30,7 +30,17 @@ interface Transformer<OptionType = unknown> {
    * If V8 coverage is _not_ active, and this is `false`. Jest will instrument the code returned by this transformer using Babel.
    */
   canInstrument?: boolean;
-  createTransformer?: (options?: OptionType) => Transformer;
+
+  /**
+   * Create transformer function:
+   *
+   * `cacheFS`: if custom transformers do module resolution and read file, custom transformers should populate this
+   * cache so that jest will avoid read the same file again hence improves performance.
```

Comment:
```suggestion
   * Creates transformer function.
   *
   * If a transformer does module resolution and reads files, it should populate `cacheFS` so that 
   * Jest avoids reading the same file again, improving performance. `cacheFS` stores entries of 
   * <file path, file contents>
```

Reply:
done 👍